django/contrib/postgres/fields/array.py

```diff
@@ -186,7 +201,7 @@ def as_sql(self, qn, connection):
         return sql, params
 
 
-@ArrayField.register_lookup
+@ArrayField.register_transform
 class ArrayLenTransform(Transform):
```

Comment:
If `Transform` is now just a back-compat shim (presumably to be deprecated at some point?) shouldn't we remove uses of it within Django?